data/src/main/java/com/example/data/service/responses/CloudResponse.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.data.service.responses
+
+import com.example.data.Utils.ConstantUtils
+import com.google.gson.annotations.SerializedName
+
+data class CloudResponse(
+    @SerializedName("all")
+    val all: Int = ConstantUtils.DEFAULT_INT
```

Comment:
var names don't need to be equal to serialized names

Reply:
Done